.github/workflows/dev-deploy-orch-stub-sp.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build and deploy Orchestration stub
+run-name: Build and deploy Orchestration stub to ${{ inputs.environment }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment to run against"
+        type: environment
+        required: true
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./orchestration-stub
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    environment: ${{ inputs.environment }}
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
```

Comment:
We don't run any sonar analysis at the moment, do we?

Reply:
We don't yet. That was a copy-paste stowaway from another repo with similar workflow. Removed the comments now.